Updated bootstrap template

ralgozino opened the pull request:
Updated bootstrap template emulating the default layout as in "pale_blue" with current upstream master rebase.

jieter commented:
@ralgozino thanks for the pull,  I'll add some line comments.

Also: CI fails, can you look into that?

jieter commented on django_tables2/templates/django_tables2/bootstrap.html:
```diff
@@ -1,11 +1,12 @@
 {% load querystring from django_tables2 %}
-{% load i18n %}
+{% load l10n %}
```
[table.html has i18n here](https://github.com/bradleyayers/django-tables2/blob/master/django_tables2/templates/django_tables2/table.html#L2).

jieter commented on django_tables2/templates/django_tables2/bootstrap.html:
```diff
@@ -19,18 +20,19 @@
                     {% endfor %}
                 </tr>
                 </thead>
+            {%endif%}
```
Add spaces to be consistant with the surrounding code.

ralgozino commented:
Thanks for the feedback @jieter 

- I've fixed the line comments and I'm currently looking into the CI failures.

- I don't like having the totals like '10 o 33 jobs' either; I've removed them from both table.html and bootstrap.html.

- About the custom class attribute, I will comment on the issue.

- I Will update the screenshots.

jieter commented:
Thanks for adding the changes. CI still fails...

jieter commented on django_tables2/templates/django_tables2/bootstrap.html:
```diff
@@ -1,42 +1,40 @@
 <div class="table-container">
     {% block table %}
-        <table class="table table-bordered table-striped"{% if table.attrs %} {{ table.attrs.as_html }}{% endif %}>
+        <table {% if table.attrs %} {{ table.attrs.as_html }}{%else%}class="table"{% endif %}>
```
Please add spaces after and before %.

jieter commented on django_tables2/templates/django_tables2/bootstrap.html:
```diff
@@ -55,26 +53,36 @@
 
     {% if table.page and table.paginator.num_pages > 1 %}
+        {% with table.page.paginator.count as total %}
+        {% with table.page.object_list|length as count %}
```
Aren't these covered by the `blocktrans`? on line 71?

jieter commented on django_tables2/templates/django_tables2/semantic.html:
```diff
@@ -0,0 +1,86 @@
+
+<div class="ui container table-container">
+    {% block table %}
+        <table {% if table.attrs %} {{ table.attrs.as_html }}{%else%}class="ui celled table"{% endif %}>
```
Another space issue.

jieter commented on example/app/views.py:
```diff
@@ -77,6 +78,24 @@ def bootstrap(request):
+def semantic(request):
+    '''Demonstrate the use of the semantic template'''
+    # create some fake data to make sure we need to paginate
+    if Person.objects.all().count() < 50:
```
This block occurs twice now, can you DRY it up?

jieter commented on example/templates/semantic_template.html:
```diff
@@ -0,0 +1,22 @@
+    <h3>django_tables2 with semantic template example</h3>
+    <div class="ui container">
+
+<!--
```
Why is this commented out? Shouldn't you include the stylesheet you added?

jieter commented on django_tables2/static/django_tables2/semantic.css:
```diff
@@ -0,0 +1,8 @@
+.table-container th.desc:after {
+    content: '\0000a0\0025bc';
+    float: right;
+}
```
This is exactly the same as the bootstrap CSS, is that correct?

ralgozino replied:
I added the CSS following the bootstrap one to see if it was needed. I've dropped it since is not the case.

ralgozino commented:
I'm pushing some corrections, but unfortunately I can't make tox work due to a bug with virtualbox and symlinks in shared folders. I'm trying to fix it in order to fix the test that are fialing. Sorry about that.

ralgozino commented:
@jieter I hope to have addressed all of your comments. Let me know what you think.

Cheers!

jieter commented:
Finally time to look at this again. Thanks, merged.
